ktor-client/ktor-client-jetty-jakarta/jvm/src/io/ktor/client/engine/jetty/jakarta/JettyResponseListener.kt

```
@@ -117,31 +133,19 @@ internal class JettyResponseListener(
     @OptIn(DelicateCoroutinesApi::class)
     private fun runResponseProcessing() = GlobalScope.launch(callContext) {
         while (true) {
-            val (buffer, callback) = backendChannel.receiveCatching().getOrNull() ?: break
+            val (buffer) = backendChannel.receiveCatching().getOrNull() ?: break
             try {
                 if (buffer.remaining() > 0) channel.writeFully(buffer)
-                callback.succeeded()
             } catch (cause: ClosedWriteChannelException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)